### DISCUSSION TITLE
The comment // coverage:ignore-file not present in generated files

### DISCUSSION THREAD

**Discussion by sinhaaritro** [rrousselGit/riverpod#3605] [Q&A]
Hi, I noticed that .g files generated by Riverpod don't have the comment `// coverage:ignore-file` to ignore from test coverage. I am using the class-based providers.
Is there a reason for it?
I saw that the generated files from freezed package had the line. So is there a reason for not having the ignore coverage line in Riverpod?

**Comment by TonyD-TuftAndPaw** [comment]
There's a `flutter test --coverage` arg [in the master branch](https://github.com/dart-lang/coverage/pull/496) (not in stable yet) that we can use to hide the generated files. But then a separate solution is required for Github actions (for example).

Adding `// coverage:ignore-file` is probably best, in both the generated files and the future meta programming files too.
